src/pages/calculator.tsx

```diff
@@ -0,0 +1,386 @@
+import FinalAmount from '@components/calculator/FinalAmount';
+import CTA from '@components/CTA';
+import {
+	FormInput,
+	FormInputWrapper,
+	FormRadioGroup,
+	FormRadioLabel,
+	FormRadioWrapper,
+	FormSelect,
+} from '@components/styled/Forms';
+import Link from 'next/link';
+import { useState } from 'react';
+import styled from 'styled-components';
+import {
+	calculateCylinderResinInKg,
+	calculateRectangularResinInKg,
+} from 'utils/calculator';
+
+export enum PRODUCTTYPE {
```

Comment:
Please remove all these types export and take them somewhere else, maybe a separate `data` file or so

Reply:
`FinalAmount.tsx` would ba a better option

Reply:
Sure done